src/dataviews/histogram-dataview/histogram-data-model.js

```diff
@@ -32,6 +33,7 @@ module.exports = Model.extend({
   },
 
   initialize: function () {
+    this.sync = BackboneCancelSync.bind(this);
```

Comment:
This has been added to a data model, not a dataview model. `Search model` does the same thing, applying the abort sync, so for those two it could make sense to set a `data-model-base`module as in dataviews.

I wouldn't add this abort behaviour blindly to another places if we don't know if they're suffering from the same problem. If ain't broke...